src/basic/calendarspec.c

```diff
@@ -876,6 +884,53 @@ static int parse_calendar_time(const char **p, CalendarSpec *c) {
         return r;
 }
 
+static int is_valid_timezone(char *tz, bool *result)
+{
```

Comment:
nitpick, please follow coding style: opening bracket for functions on same line as function name please